src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  FC, useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { useParams } from 'react-router-dom';
+import { TodoHeader } from '../TodoHeader';
+import { TodoList } from '../TodoList';
+import { TodoFooter } from '../TodoFooter';
+import { ErrorNotification } from '../ErrorNotification';
+import { Todo } from '../../types/Todo';
+import { TodoStatusFilter } from '../../types/TodoStatusFilter';
+import {
+  createTodo, deleteTodo, getTodos, updateTodo,
+} from '../../api/todos';
+import {
+  getActiveTodos, getCompletedTodos, getVisibleTodos, validateTitle,
+} from '../../utils/helpers';
+
+const USER_ID = 10684;
+
+export const Layout: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loadingTodos, setLoadingTodos] = useState<number[]>([0]);
```

Comment:
The problem with `0` here is that it's a wholly hardcoded value and shouldn't be used like that because whoever will read this code would need context to understand what's happening here. Think about a workaround